### PR TITLE
feat(FOROME-1300): make datasets unfolded in the beggining

### PR DESCRIPTION
**Pull request by QSdmitrioul**
[Task](https://quantori.atlassian.net/browse/FOROME-1300)

### CODE REVIEW

File: src/store/dirinfo.ts [hunk 1]
```diff
@@ -26,6 +26,8 @@ class DirInfoStore {
   iframeInfoFullscreen = false
   activeInfoName = ''
 
+  private readonly closedDatasets = new Set<string>()
```
Comment (AndreyUstyumenko) [inline src/store/dirinfo.ts]: folded

File: src/store/dirinfo.ts [hunk 2]
```diff
@@ -36,6 +38,18 @@ class DirInfoStore {
     makeAutoObservable(this)
   }
 
+  public closeDs(name: string) {
```
Comment (AndreyUstyumenko) [inline src/store/dirinfo.ts]: fold

File: src/store/dirinfo.ts [hunk 3]
```diff
@@ -36,6 +38,18 @@ class DirInfoStore {
...
+    this.closedDatasets.add(name)
+  }
+
+  public openDs(name: string) {
```
Comment (AndreyUstyumenko) [inline src/store/dirinfo.ts]: unfold

File: src/store/dirinfo.ts [hunk 4]
```diff
@@ -36,6 +38,18 @@ class DirInfoStore {
...
+    this.closedDatasets.delete(name)
+  }
+
+  public isDsClosed(name: string): boolean {
```
Comment (AndreyUstyumenko) [inline src/store/dirinfo.ts]: folded

File: src/store/dirinfo.ts (hunk 1 above)
Comment (QSdmitrioul) [reply src/store/dirinfo.ts]: Done

File: src/store/dirinfo.ts (hunk 3 above)
Comment (QSdmitrioul) [reply src/store/dirinfo.ts]: Done

File: src/store/dirinfo.ts (hunk 2 above)
Comment (QSdmitrioul) [reply src/store/dirinfo.ts]: Done

File: src/store/dirinfo.ts (hunk 4 above)
Comment (QSdmitrioul) [reply src/store/dirinfo.ts]: Done

**Review by AndreyUstyumenko — APPROVED** [review]

**Review by Lapk1n — APPROVED** [review]

**Pull request merged**